### DISCUSSION TITLE
Tasmota32 without WiFi

### DISCUSSION THREAD

**Discussion by SHA-NDY** [arendst/Tasmota#19258] [Q&A] [answered]
Hi all,
I have an **ESP32** running with **Tasmota32** v12.5.0. BLE is enabled and I use it to scan a device to trigger an action when this device is present. When the ESP32 has no WiFi connection it seems to be disabled. The scanning no longer works and no device trigger will happen. As soon as WiFi is connected again everything works as expected.
Can I configure Tasmota to also work if no WiFi connection is present?

The WiFi connection gets lost sometimes because it has poor covering and is established by a repeater.

WifiConfig is set to "4" and the WiFi Network 2 is configured as a Dummy.

Thanks!

**Comment by barbudor** [comment, marked as answer]
Working as designed
Tasmota is designed to work with wifi.

**Reply by sfromis** [reply]
Or other network connection if you can connect Ethernet
